src/pages/docs/runbooks/runbook-examples/terraform/provision-aws-with-terraform.md

```diff
@@ -278,8 +278,22 @@ workerPool="#{Project.Octopus.Server.WorkerPool}"
 machinePolicy="#{Project.Octopus.Server.MachinePolicy}"
 space="#{Project.Octopus.Server.Space}"
 
-sudo apt-key adv --fetch-keys "https://apt.octopus.com/public.key"
-sudo add-apt-repository "deb https://apt.octopus.com/ stretch main"
+apt-get update && apt-get install --no-install-recommends gnupg curl ca-certificates apt-transport-https && \
```

Comment:
Should probably add a "-y" to the install command here since it's part of an automated process

Reply:
Yeah, it's probably best to add the -y, even if the previous example didn't have it for here and the tentacle install a couple of lines down.

Reply:
Got that added.